recipes/highfive/all/conanfile.py

```diff
@@ -65,6 +67,7 @@ def generate(self):
         tc.variables["HIGHFIVE_EXAMPLES"] = False
         tc.variables["HIGHFIVE_BUILD_DOCS"] = False
         tc.variables["HIGHFIVE_USE_INSTALL_DEPS"] = False
+        tc.variables["HIGHFIVE_STATIC_HDF5"] = self.options.with_static_hdf5
```

Comment:
Shouldn't this be true if `hdf5` is built statically?

```suggestion
        tc.variables["HIGHFIVE_STATIC_HDF5"] = not self.dependencies["hdf5"].options.shared
```

Reply:
I guess it depends on how we want to drive the static version of hdf5. According to 2.8.0, there exist an option in CMake to select to link to a static version of HDF5. I think this should be reflected as an options for the HighFive package. 

We should set `self.dependencies["hdf5"].options.shared = self.options.with_static_hdf5` though.

Reply:
That seems a little backwards to me, as consumers probably expect that setting `hdf5/*:shared=False` to link statically.

Reply:
Fair enough. I will change it